docs/how-to-work-on-coding-challenges.md

```diff
@@ -109,9 +109,9 @@ Optional Test tear down code.
 
 ## Numbering Challenges
 
-Every challenge needs an `id`. If you don't specify one, then MongoDB will create a new random one when it saves the data; however, we don't want it to do that, since we want the challenge ids to be consistent across different environments (staging, production, lots of different developers, etc.).
+Every challenge needs an `id`. If you don't specify one, then MongoDB will create a new random `id` when it saves the data; however, we don't want it to do that, since we want the challenge ids to be consistent across different environments (staging, production, lots of different developers, etc.).
```

Comment:
Let's break this into two sentences and simplify it a bit:

```
Every challenge needs an `id`. If you don't specify one, then MongoDB will create a new random `id` when it saves the data. However, we don't want it to do that since the challenge ids should be consistent across different environments (staging, production, lots of different developers, and so on).
```

Reply:
I can see why you'd want to separate them, but they're thoughts within the same vein, so I think they should stay connected with the semicolon. I'd be interested to see what another human thinks. 😄

Reply:
@allthedoll Please break into two sentences.  Thanks.